docs/data-sources/netflow_exporter_policy.md

```diff
@@ -0,0 +1,86 @@
+---
+# Documentation generated by "gen/generator.go"; DO NOT EDIT.
+# In order to regenerate this file execute `go generate` from the repository root.
+# More details can be found in the [README](https://github.com/CiscoDevNet/terraform-provider-aci/blob/master/README.md).
+subcategory: "Tenant Policies"
+layout: "aci"
+page_title: "ACI: aci_netflow_exporter_policy"
+sidebar_current: "docs-aci-data-source-aci_netflow_exporter_policy"
+description: |-
+  Data source for ACI Netflow Exporter Policy
+---
+
+# aci_netflow_exporter_policy #
+
+Data source for ACI Netflow Exporter Policy
+
+## API Information ##
+
+* Class: [netflowExporterPol](https://pubhub.devnetcloud.com/media/model-doc-latest/docs/app/index.html#/objects/netflowExporterPol/overview)
+
+* Supported in ACI versions: 2.2(1k) and later.
+
+* Distinguished Name Formats:
+  - `uni/infra/exporterpol-{name}`
+  - `uni/tn-{name}/exporterpol-{name}`
+
+## GUI Information ##
+
+* Locations:
+  - `Tenants -> Policies -> NetFlow -> NetFlow Exporters`
+  - `Fabric -> Access Policies -> Policies -> Interface -> NetFlow -> NetFlow Exporters`
+
+## Example Usage ##
+
+```hcl
+
+data "aci_netflow_exporter_policy" "example_tenant" {
+  parent_dn = aci_tenant.example.id
+  name      = "netfow_exporter"
+}
+
+```
+
+## Schema ##
+
+### Required ###
+
+* `parent_dn` - (string) The distinguished name (DN) of the parent object, possible resources:
+  - [aci_tenant](https://registry.terraform.io/providers/CiscoDevNet/aci/latest/docs/resources/tenant) ([fvTenant](https://pubhub.devnetcloud.com/media/model-doc-latest/docs/app/index.html#/objects/fvTenant/overview))
+  - The distinguished name (DN) of classes below can be used but currently there is no available resource for it:
+    - [infraInfra](https://pubhub.devnetcloud.com/media/model-doc-latest/docs/app/index.html#/objects/infraInfra/overview)
+
+  - Default: `uni/infra`
```

Comment:
Did you have a discussion on whether parent_dn should be in Optional in the data source docs?

Reply:
what is the result of this?

Reply:
what is the result of this?

Reply:
Just added a new Optional Section when the parent_dn has a default value.